### PR TITLE
Update version-control-basics.md

### PR DESCRIPTION
**Pull request by mirnawong1**
update gitignore content per slack thread here: https://dbt-labs.slack.com/archives/C012TTNN0JK/p1712760453549829

### CODE REVIEW

File: website/docs/docs/collaborate/git/version-control-basics.md [hunk 1]
```diff
@@ -54,24 +54,23 @@ Refer to [merge conflicts](/docs/collaborate/git/merge-conflicts) to learn how t
...
-
-Some git providers will automatically create a 'boilerplate' `.gitignore` file when the repository is created.  However, based on dbt Labs' experience, these default `.gitignore` files typically don't include the required entries for dbt Cloud to function correctly.
-
-The `.gitignore` file can include unrelated files and folders if the code repository requires it. However, the following folders must be included in the `gitignore` file to ensure dbt Cloud operates smoothly:
+dbt Cloud implements a global [`.gitignore file`](https://github.com/dbt-labs/dbt-starter-project/blob/main/.gitignore) that automatically excludes the following sub-folders from your git repository to ensure smooth operation. This global `.gitignore` is automatically applied to projects in dbt Cloud, ensuring these directories are not tracked, which is crucial for smooth dbt Cloud operation:
```
Comment (mirnawong1) [inline website/docs/docs/collaborate/git/version-control-basics.md]:
```suggestion
dbt Cloud implements a global [`.gitignore file`](https://github.com/dbt-labs/dbt-starter-project/blob/main/.gitignore) (ensuring these directories are not tracked) that automatically excludes the following sub-folders from your git repository to ensure smooth operation:
```

File: website/docs/docs/collaborate/git/version-control-basics.md [hunk 2]
```diff
@@ -54,24 +54,23 @@ Refer to [merge conflicts](/docs/collaborate/git/merge-conflicts) to learn how t
 
 ## The .gitignore file
 
-To make sure dbt Cloud runs smoothly, you must exclude certain sub-folders in your git repository containing your dbt project from being tracked by git. You can achieve this by adding three lines to a special file named [.gitignore](https://github.com/dbt-labs/dbt-starter-project/blob/main/.gitignore). This file is placed in the root folder of your dbt project.
-
-Some git providers will automatically create a 'boilerplate' `.gitignore` file when the repository is created.  However, based on dbt Labs' experience, these default `.gitignore` files typically don't include the required entries for dbt Cloud to function correctly.
-
-The `.gitignore` file can include unrelated files and folders if the code repository requires it. However, the following folders must be included in the `gitignore` file to ensure dbt Cloud operates smoothly:
+dbt Cloud implements a global [`.gitignore file`](https://github.com/dbt-labs/dbt-starter-project/blob/main/.gitignore) (ensuring these directories are not tracked) that automatically excludes the following sub-folders from your git repository to ensure smooth operation:
```
Comment (mirnawong1) [inline website/docs/docs/collaborate/git/version-control-basics.md]:
```suggestion
dbt Cloud implements a global [`.gitignore file`](https://github.com/dbt-labs/dbt-starter-project/blob/main/.gitignore) that automatically excludes the following sub-folders from your git repository to ensure smooth operation:
```

File: website/docs/docs/collaborate/git/version-control-basics.md [hunk 3]
```diff
@@ -54,24 +54,23 @@ Refer to [merge conflicts](/docs/collaborate/git/merge-conflicts) to learn how t
...
 ```
 dbt_packages/
 logs/
 target/
 ```
 
-**Note** &mdash; By using a trailing slash, these lines in the `gitignore` file serve as 'folder wildcards', excluding all files and folders within those folders from being tracked by git. 
+This inclusion uses a trailing slash, making these lines in the `.gitignore` file act as 'folder wildcards' that prevent any files or folders within them from being tracked by git. You can also specify additional exclusions as needed for your project.
+
+However, this global `.gitignore` _does not_ apply to dbt Core and dbt Cloud CLI users directly. Therefore, if you're working with dbt Core or dbt Cloud CLI, you need to manually add these three lines to your project's `.gitignore` file.
```
Comment (mirnawong1) [inline website/docs/docs/collaborate/git/version-control-basics.md]:
```suggestion
However, this global `.gitignore` _does not_ apply to dbt Core and dbt Cloud CLI users directly. Therefore, if you're working with dbt Core or dbt Cloud CLI, you need to manually add the three lines mentioned previously to your project's `.gitignore` file.
```

**Review by StefanDBTLabs — APPROVED** [review]

**Comment by mirnawong1** [comment]
thank you @StefanDBTLabs !

**Pull request merged**